account_invoice_force_number/tests/test_account_force_number.py

```diff
@@ -4,49 +4,43 @@
 
 
 from odoo.exceptions import UserError
-from odoo.tests.common import TransactionCase
 
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+class TestAccountForceNumber(AccountTestInvoicingCommon):
+    def create_invoice(self, invoice_amount=450, move_name=None):
```

Comment:
Considering that in your tests you always use the default value of `invoice_amount`, I would avoid this parameter.